gammapy/data/observations.py

```diff
@@ -503,6 +504,49 @@ def write(self, path, overwrite=False, format="gadf", include_irfs=True):
 
         hdul.writeto(path, overwrite=overwrite)
 
+    def copy(self, in_memory=False, **kwargs):
+        """Copy observation
+        
+        Parameters
+        ----------
+        in_memory : bool
+            Copy observation in memory
```

Comment:
Maybe add "set True" to overwrite parameters?

Reply:
Thanks, done!